modules/smtp/site.js

```diff
@@ -375,3 +389,25 @@ function smtpSettingsPageHandler() {
         );
     });
 }
+
+$(function() {
+    if (hm_is_logged()) {
+        var scheduled_msg_count = 0;
+        var sendScheduledMessages = function() { 
```

Comment:
I have mentioned this many times, and I’ll repeat it: you should avoid using `var` and instead choose between `const` and `let`.